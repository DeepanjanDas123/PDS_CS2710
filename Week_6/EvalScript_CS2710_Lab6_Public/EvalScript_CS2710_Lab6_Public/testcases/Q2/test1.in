7
NEW PUSH_BEGIN 5 PUSH_BEGIN 6 PUSH_BEGIN 8 PUSH_BEGIN 9 PUSH_BEGIN 14 REVERSE 3
