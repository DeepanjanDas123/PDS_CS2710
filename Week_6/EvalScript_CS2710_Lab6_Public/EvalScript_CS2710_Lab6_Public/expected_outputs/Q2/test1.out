8 9 14 6 5
